examples/delegator-macro/src/lib.rs

```diff
@@ -0,0 +1,18 @@
+use near_abi_client_macros::near_abi_client;
+use workspaces::network::DevAccountDeployer;
+
+near_abi_client! { "src/adder.json" }
```

Comment:
I wonder if it would be more clear if we indicate somehow what the output struct is by having something like:

`type ExtAbi = near_abi_client!...`

Or some other mechanism within the macro to indicate the output type. Otherwise, I'm not sure it will be intuitive to developers what the output is.

Reply:
I don't think I can do specifically `type ExtAbi = near_abi_client!...` because Rust heavily restricts what can be on the right side of `type T =`. But I changed this to look like `near_abi_client! { type AbiClient for "src/adder.json" }` which is hopefully reasonable?